apps/web/src/components/Publication/Poll/Choices.tsx

```diff
@@ -56,7 +56,7 @@ const Choices: FC<ChoicesProps> = ({ poll, refetch }) => {
       Leafwatch.track(PUBLICATION.WIDGET.POLL.VOTE, { poll_id: id });
       toast.success('Your poll has been casted!');
     } catch {
-      toast.error(Errors.SomethingWentWrong);
+      toast.error(Errors.PollExpired);
```

Comment:
there might be multiple reason for failure, we need to maintain some kind of code on backend!